.gitignore

```diff
@@ -0,0 +1,6 @@
+venv/
+*.sqlite
+*.sqlite3
+*.sql
+*.sql.gz
+__pycache__/
```

Comment:
Add Eof(End-of-file), need to add a blank line at the end
![image](https://github.com/user-attachments/assets/5c3f2dd3-6179-40d4-97fc-f5cef151f0d8)